pallets/xanchor/src/tests.rs

```diff
@@ -234,7 +275,7 @@ fn should_bridge_anchors_using_xcm() {
 	});
 
 	ParaB::execute_with(|| {
-		setup_environment(Curve::Bn254);
+		//setup_environment(Curve::Bn254);
```

Comment:
Is commenting this out intentional or can it be removed?

Reply:
I was trying to check something out. I have reverted now.